saleor/giftcard/models.py

```diff
@@ -38,7 +39,9 @@ def active(self, date):
 
 
 class GiftCard(ModelWithMetadata):
-    code = models.CharField(max_length=16, unique=True, db_index=True)
+    code = models.CharField(
+        max_length=16, unique=True, validators=[MinLengthValidator(8)], db_index=True
```

Comment:
Are we sure that we want to introduce `MinLengthValidator` in this field? If we release it to 3.7 it will be a breaking change in path release.